Explorer/Assets/DCL/Infrastructure/Global/Dynamic/RealmLaunchSettings.cs

```diff
@@ -26,6 +28,7 @@ public struct PredefinedScenes
 
         [SerializeField] internal InitialRealm initialRealm;
         [SerializeField] internal Vector2Int targetScene;
+        [SerializeField] internal bool ForceHomePosition = false;
```

Comment:
Do you need it? If editor position is not forced, shouldnt this be always `true`?

Also, I dont quite get how this works in build, since by default its `false`.